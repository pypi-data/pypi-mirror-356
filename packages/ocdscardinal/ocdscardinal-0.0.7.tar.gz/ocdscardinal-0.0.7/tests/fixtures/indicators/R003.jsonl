{"ocid":"open","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-16T17:59:59Z"},"procurementMethod":"open"}}
{"ocid":"selective","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-16T17:59:59Z"},"procurementMethod":"selective"}}
{"ocid":"limited","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-16T17:59:59Z"},"procurementMethod":"limited"}}
{"ocid":"direct","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-16T17:59:59Z"},"procurementMethod":"direct"}}
{"ocid":"missing","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-16T17:59:59Z"}}}
{"ocid":"exact","tender":{"tenderPeriod":{"startDate":"2000-01-01T18:00:00Z","endDate":"2000-01-16T18:00:00Z"}}}
{"ocid":"invalid start date","tender":{"tenderPeriod":{"startDate":"invalid","endDate":"2000-01-01T00:00:00Z"}}}
{"ocid":"invalid end date","tender":{"tenderPeriod":{"startDate":"2000-01-01T00:00:00Z","endDate":"invalid"}}}
