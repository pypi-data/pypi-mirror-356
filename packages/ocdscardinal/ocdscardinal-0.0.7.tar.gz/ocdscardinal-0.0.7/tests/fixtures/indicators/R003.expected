{"OCID":{"open":{"R003":1.0},"selective":{"R003":1.0},"limited":{"R003":1.0},"direct":{"R003":1.0},"missing":{"R003":1.0}}}
